pkgs/applications/misc/styx/default.nix

```diff
@@ -15,27 +13,18 @@ stdenv.mkDerivation rec {
 
   server = "${caddy}/bin/caddy";
   linkcheck = "${linkchecker}/bin/linkchecker";
+  nixpkgs = pkgs.path;
 
   nativeBuildInputs = [ asciidoctor ];
 
-  outputs = [ "out" "lib" "themes" ];
-
-  propagatedBuildInputs = [
-    file
-    lessc
-    sass
-    asciidoctor
-    multimarkdown
-    perlPackages.ImageExifTool
-    python3Packages.parsimonious
-  ];
+  outputs = [ "out" "themes" ];
 
   installPhase = ''
     mkdir $out
-    install -D -m 777 src/styx.sh $out/bin/styx
 
-    mkdir -p $out/share/styx-src
-    cp -r ./* $out/share/styx-src
+    install -D -m 777 src/styx.sh $out/bin/styx
+    cp src/default.nix $out/default.nix
+    cp src/styx-config.nix $out/styx-config.nix
```

Comment:
Why do we need those files in the final package at ``$out/`` ?

Reply:
It is used in the upstream `derivation.nix` 
Why it was (recently) introduced..Good question

Reply:
They are part of the secondary eval loop that styx does internally, similar to how `colmena` ships with `/hive/eval.nix` (although as constant as part of the rust code).